components/transaction/internal/adapters/http/in/transaction_creation_helpers.go

```
@@ -257,6 +243,452 @@ func (handler *TransactionHandler) BuildOperations(
+		Amount:          operation.Amount{Value: &amt.Value},
+		Balance:         debitBalance,
+		BalanceAfter:    debitBalanceAfter,
+		BalanceID:       blc.ID,
+		AccountID:       blc.AccountID,
+		AccountAlias:    pkgTransaction.SplitAlias(blc.Alias),
+		BalanceKey:      blc.Key,
+		OrganizationID:  blc.OrganizationID,
+		LedgerID:        blc.LedgerID,
+		CreatedAt:       transactionDate,
+		UpdatedAt:       time.Now(),
+		Route:           ft.Route,
```

golangci-lint (2.11.3)

[error] SA1019: ft.Route is deprecated: legacy route identifier, use routeId instead. Contains the same value as routeId but as a free-form string. 

(staticcheck)